[Merged by Bors] - Add SSH tunneling in Lighthouse UI Siren (#4328)
sigp/lighthouse: book/src/ui-configuration.md

This allows you to enter the address and ports of the associated Lighthouse
Beacon node and Lighthouse Validator client.

> The Beacon Node must be run with the `--gui` flag set. 

> If you run Siren in the browser (by entering `localhost` in the browser), you will need to allow CORS in the http API. This can be done by adding the flag `--http-allow-origin "*"` in both beacon node and validator client. If you would like to access Siren beyond the local computer, we recommend to use ssh tunnel. This requires ssh tunnel to connect to 3 ports: `80` (assuming the port is unchanged as per the [installation guide](./ui-installation.md#docker-recommended), `5052` (for beacon node) and `5062` (for validator client). You can use the command below to perform ssh tunneling:

Reviewer: ```suggestion
If you run Siren in the browser (by entering `localhost` in the browser), you will need to allow CORS in the HTTP API. This can be done by adding the flag `--http-allow-origin "*"` for both beacon node and validator client. If you would like to access Siren beyond the local computer, we recommend using an SSH tunnel. This requires a tunnel for 3 ports: `80` (assuming the port is unchanged as per the [installation guide](./ui-installation.md#docker-recommended), `5052` (for beacon node) and `5062` (for validator client). You can use the command below to perform SSH tunneling:
```

```bash

> ssh -L 80:127.0.0.1:80 -L 5052:127.0.0.1:5052 -L 5062:127.0.0.1:5062 username@local_ip

Reviewer: ```suggestion
ssh -N -L 80:127.0.0.1:80 -L 5052:127.0.0.1:5052 -L 5062:127.0.0.1:5062 username@local_ip
```

I usually run tunnel commands with `-N` so that they don't get confused with ordinary shell sessions. If we recommend this we might want to note that the connection appears to hang

Author: Updated with some remarks about connecting with option -N, thank you

```  

> where `username` is the username of the server and `local_ip` is the local IP address of the server. Note that with the `-N` option in an SSH session, you will not be able to execute commands in the CLI to avoid confusion with ordinary shell sessions. The connection will appear to be "hang" upon a successful connection, but that is normal. Once you have successfully connected to the server via SSH tunneling, you should be able to access Siren by entering `localhost` in a web browser. 

Reviewer: ```suggestion
where `username` is the username of the server and `local_ip` is the local IP address of the server. Note that with the `-N` option in an SSH session, you will not be able to execute commands in the CLI to avoid confusion with ordinary shell sessions. The connection will appear to be "hung" upon a successful connection, but that is normal. Once you have successfully connected to the server via SSH tunneling, you should be able to access Siren by entering `localhost` in a web browser. 
```
